examples/ConfigurationStoreBackup/BackupAppConfigurationStore.cs

```diff
@@ -0,0 +1,184 @@
+            }
+
+            string primaryStoreEndpoint = Environment.GetEnvironmentVariable(PrimaryConfigStoreEndpointEnvVarName);
+            if (string.IsNullOrEmpty(primaryStoreEndpoint))
+            {
+                throw new InvalidOperationException($"Please ensure that the environment variable '{PrimaryConfigStoreEndpointEnvVarName}' is set to the endpoint of the primary App Configuration store.");
+            }
+
+            string secondaryStoreEndpoint = Environment.GetEnvironmentVariable(SecondaryConfigStoreEndpointEnvVarName);
+            if (string.IsNullOrEmpty(secondaryStoreEndpoint))
+            {
+                throw new InvalidOperationException($"Please ensure that the environment variable '{SecondaryConfigStoreEndpointEnvVarName}' is set to the endpoint of the secondary App Configuration store.");
+            }
+
+            QueueClient queueClient = new QueueClient(storageConnectionString, QueueName);
+            ConfigurationClient primaryAppConfigClient = new ConfigurationClient(new Uri(primaryStoreEndpoint), new ManagedIdentityCredential());
+            ConfigurationClient secondaryAppConfigClient = new ConfigurationClient(new Uri(secondaryStoreEndpoint), new ManagedIdentityCredential());
+            
+            await BackupAppConfigurationStoreAsync(queueClient, primaryAppConfigClient, secondaryAppConfigClient, log);
+        }
+
+        private static async Task BackupAppConfigurationStoreAsync(QueueClient queueClient,
+                                                                   ConfigurationClient primaryAppConfigClient,
+                                                                   ConfigurationClient secondaryAppConfigClient,
+                                                                   ILogger log)
+        {
+            // Peek to see if there are events in the queue.
+            while ((await queueClient.PeekMessagesAsync()).Value.Length > 0)
+            {
+                Response<QueueMessage[]> retrievedMessages = await queueClient.ReceiveMessagesAsync(MaxMessagesToRead);
+
+                // Extract list of key+labels from event data.
+                HashSet<KeyLabel> updatedKeyLabels = ExtractKeyLabelsFromEvents(retrievedMessages.Value, log);
+
+                // If there are any valid App Configuration events, update secondary store.
+                if (updatedKeyLabels.Count > 0)
+                {
+                    await BackupKeyValuesAsync(updatedKeyLabels, primaryAppConfigClient, secondaryAppConfigClient, log);
+                }
+
+                // Delete this batch of events from storage queue.
+                foreach (QueueMessage message in retrievedMessages.Value)
+                {
+                    try
+                    {
+                        await queueClient.DeleteMessageAsync(message.MessageId, message.PopReceipt);
+                        log.LogDebug($"Sucessfully deleted message from queue. Message ID: {message.MessageId}");
```

Comment:
Should this be `LogInformation`? Can you please make sure we are consistent in other places as well?